edb/repl/__init__.py

```diff
@@ -541,6 +544,7 @@ def _command_describe_object(
             )
         except edgedb.EdgeDBError as exc:
             render.render_exception(self.context, exc)
+            self._command_semicolon_hint(arg)
```

Comment:
Drop this

Reply:
Ah, alright, just read your remark re `InvalidReferenceError` -- I think `InvalidReferenceError` needs to be handled with a dedicated `except InvalidReferenceError` clause and a nice error message.  And a hint if `;` was there.

Reply:
OK

Reply:
This line is no longer needed since you handle it now in the `except edgedb.InvalidReferenceError as exc` above.